### ISSUE TITLE
✅ 🔗 🚀 Feature Request: Full Content Skin of Post and Archive Widgets doesn't support pages built with an Elementor template (Duplicate of #4440)

### ISSUE THREAD

**Issue by SaltedLevity** [elementor/elementor#11528]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
I'm running into an issue where I wanted to list posts on another page of the site using the "Full Content" skin selection to completely reproduce the entirety of the page's content. However if said post/page/custom post type is using an Elementor template, rather than the page being manually edited, none of the designed content appears in the list.

For clarity, by Elementor page template I mean having a designed template set with display conditions to appear on all pages/certain pages/ posts / types / etc.
It does not happen, for example, if I save a layout to my templates and insert that into an edited page (like a section or full page layout inserted). Pages that were individually designed/designed using inserted template elements show in their entirity.

## Steps to reproduce

1) Create a site template design. E.g. a design for singular posts. Set display conditions accordingly.
2) Create a post that meets the above display condition criteria
3) On another page, insert a "Posts" item widget. Set the skin to "Full Content"
-- The same issue occurs if you try to use an Archive list, or create an archive view of the pages
4) List will not show featured images, nor any non-plain content, nor any custom fields.

To put it in terms of a more practical use-case (using a custom post type in my example, however I tested it and this applies to standard posts!)
Say I created a custom post type for Recipes (using CPT). In there I had some custom fields (using ACF) for ingredients and steps to make, as well as using the standard input for a general description of the recipe.
Then I make a "Recipe" template in elementor that uses these custom fields to populate the display and each individual recipe page looks great.
Then I made a custom page that was all dedicated to cakes, and wanted to insert all of my cake recipes right there on the page using the same nice template design they already have. Using the posts widget, none of the important parts of my recipe are included. In the full content view it doesn't even pull in the featured image, nor would it pull in the ingredients or steps. So in essence I can't reproduce my recipes here or in an archive in a meaningful way.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
System Info Paste Bin: https://pastebin.com/5FvJJ82A
</details>

**Comment by TomRaraAvis** [comment]
@SaltedyLevity I think this would request would solve your issues? #12059

**Comment by SaltedyLevity** [comment]
I've actually been using the other plugin, ele custom skins, since a bit after I made this post. However I would significantly prefer if elementor natively supported it!

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce the new Loop Builder released with Elementor Pro v3.8.0! 🥳 

With the new Loop Grid widget, you can customize your post listings with custom templates! 💪 
Drag & Drop widgets to create complex designs, fine-tune your Loop Grid layout, add alternate templates, and much more!

This is just the first phase. More customization is coming!

Check out the article here: https://elementor.com/help/loop-grid/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
